### DISCUSSION TITLE
React Table Component Gridlines Overridden by Bootstrap

### DISCUSSION THREAD

**Discussion by Commando-Brando** [primefaces/community#1387] [PrimeReact]
We are migrating away from bootstrap to prime react.
I am trying to use the table component with grid lines but I know the boostrap import is what is preventing it but I cannot remove it currently.

```tsx
import "primereact/resources/themes/lara-light-indigo/theme.css";
import "primereact/resources/primereact.min.css";
import "primeicons/primeicons.css";
import "primeflex/primeflex.css";
import "./assets/css/custom-prime.css";

// Custom CSS
import "./App.css";
import "bootstrap/dist/css/bootstrap.min.css";
```

I have tried reorganized CSS imports but it does not seem to make a difference.

**Comment by melloware** [comment]
Hmmm you might have to just use a custom style to override bootstrap's style by making you gridlines style more specific.
